Update audio_transcription.py

RafalKaufman opened the pull request:
Modified the variable and method names in order to keep them 'self explanatory'.

RafalKaufman committed:
Update audio_transcription.py

Modified the variable and method names in order to keep them 'self explanatory'.

MichalKapala commented:
@RafalKaufman 
![image](https://github.com/RafalKaufman/SpeechSummarizer/assets/35117419/2b260c25-020d-41f2-ae0e-94e04ccfd017)

RafalKaufman commented:
approve needed my lord

MichalKapala commented on audio_transcription.py:
```diff
@@ -5,46 +5,47 @@
-            audio_listened = self.recognizer.record(source)
-            text = self.recognizer.recognize_google(audio_listened)
-        return text
+    def transcribe(self, audio_file_name):
```
As we are improving these code, I got one more improvement suggestion 

```suggestion
    def transcribe(self, audio_file_name : str) -> str:
```

As python language does not support strong typing, it's good to make such suggestion.
PS: Applicable also for other methods

MichalKapala commented on audio_transcription.py:
```diff
@@ -5,46 +5,47 @@
+                chunk_transcription = f"{chunk_transcription.capitalize()}. "
+                whole_transcription += chunk_transcription
+        return whole_transcription
     def get_dialogue_transcription(self):
```
In general the `get` method  tells user that he should receive something from that method. I suggest to return self.transcription in this method and don't relay only on magic __str__ method

MichalKapala commented:
@RafalKaufman Good improvements :+1:  Please be aware that my comment's are only suggestion and if you have some doubts feel free to start discussion ;)

RafalKaufman committed:
Update audio_transcription.py relative to comments

Returned value in get_dialogue_transcription method and added method type suggestions.

RafalKaufman commented:
@MichalKapala appreciate your comments. I've updated the file again. Could you spare a moment to give a **final review**?

MichalKapala commented:
> @MichalKapala appreciate your comments. I've updated the file again. Could you spare a moment to give a **final review**?

I'd love to, but first check the code cause i see there is some syntax problem ;)

RafalKaufman committed:
Finalize audio_transcription.py

RafalKaufman committed:
Finalize audio_transcription.py
